src/main/scala/mesosphere/marathon/core/launchqueue/impl/LaunchQueueDelegate.scala

```diff
@@ -33,13 +37,18 @@ private[launchqueue] class LaunchQueueDelegate(
   override def add(app: AppDefinition, count: Int): Unit = askQueueActor("add")(LaunchQueueDelegate.Add(app, count))
 
   private[this] def askQueueActor[T](method: String)(message: T): Any = {
+    val answerFuture: Future[Any] = askQueueActorFuture(method)(message)
+    Await.result(answerFuture, config.launchQueueRequestTimeout().milliseconds)
+  }
+
+  def askQueueActorFuture[T](method: String)(message: T): Future[Any] = {
```

Comment:
can the scope also be private[this] (according to the code it can) or is there any reason for that?